### PR TITLE
feat: add feature to expose yaml format OpenAPI document

### PR DESCRIPTION
**Pull request by starcharles**
**Describe the PR**
add feature to expose yaml format OpenAPI document.

You can see yaml formatted openapi document at "/swagger/doc.yaml" endpoint.

**Relation issue**
https://github.com/swaggo/echo-swagger/issues/109

**Additional context**
It's sometimes comfortable to get yaml format openapi document from Swagger UI.

### CODE REVIEW

**Review by EdouardBavoux — COMMENTED** [review]
Thanks a lot. I just have one suggestion so far. I'll test it and let you know

**Review by EdouardBavoux — APPROVED** [review]
OK, I did a few tests and it looks good to me ! Thanks

**Comment by starcharles** [comment]
Thanks fo approval this PR!
Is this PR can be merged?
or another reviewer need to approve?

**Comment by starcharles** [comment]
@ubogdan Please review this PR if you have time Thanks.

File: swagger.go [hunk 1]
```diff
@@ -170,7 +173,20 @@ func EchoWrapHandler(options ...func(*Config)) echo.HandlerFunc {
 			}
 
 			_, _ = c.Response().Writer.Write([]byte(doc))
+		case "doc.yaml":
```
[the remark this answers is not in the document]
Comment (ubogdan) [reply swagger.go:176]: The yaml file is not embedded in the binary so reading it from disk may be a problem,

**Review by ubogdan — APPROVED** [review]

File: swagger.go (hunk 1 above)
[the remark this answers is not in the document]
Comment (EdouardBavoux) [reply swagger.go:176]: Makes perfect sense, thanks for the info. So let's merge it this way.

**Comment by starcharles** [comment]
need tests?

**Comment by ubogdan** [comment]
@starcharles just got bussy with other things.

**Pull request merged**